PKGBUILD

```diff
@@ -75,11 +77,16 @@ prepare() {
     patch -Np1 < "../$src"
   done
 
-  echo "Installing rust-src"
-  rustup component add rust-src
```

Comment:
Maybe we could detect if `rustup` is available and then call `rustup component add rust-src`.

Reply:
This way should work, I'd think